### PR TITLE
Add Firebase Test Lab

### PR DESCRIPTION
**Pull request by tiembo**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,10 +36,43 @@ jobs:
...
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/plaid
+      - run:
+          name: Setup Google Cloud auth
+          command: scripts/ftl_setup.sh
+      - run:
+          name: Run tests on Firebase Test Lab and download results
+          command: echo "TODO"
```
Comment (keyboardsurfer) [inline .circleci/config.yml]: `"TODO: Integrate with Firebase Test Lab"`

File: scripts/ftl_run_tests.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    mkdir -p "$TEST_DIR/$RESULTS_DIR"
+
+	# Pull down test results
+	gsutil -m cp -r -U "gs://cloud-test-android-devrel-ci/$RESULTS_DIR/*" "$TEST_DIR/$RESULTS_DIR"
+}
+
+test_apk \
+	"app" \
+	"app/build/outputs/apk/debug/plaid-debug.apk" \
+	"app/build/outputs/apk/androidTest/debug/plaid-debug-androidTest.apk"
```
Comment (keyboardsurfer) [inline scripts/ftl_run_tests.sh]: Newline at EOF missing

File: scripts/ftl_setup.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+echo $GCLOUD_SERVICE_KEY | base64 -di > client_secret.json
+
+# Set project ID
+gcloud config set project android-devrel-ci
+
+# Auth account
+gcloud auth activate-service-account plaid-ftl@android-devrel-ci.iam.gserviceaccount.com --key-file client_secret.json
+
+# Delete secret
+rm client_secret.json
```
Comment (keyboardsurfer) [inline scripts/ftl_setup.sh]: Newline at EOF missing

File: scripts/ftl_setup.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Check if service key environment variable is set; exit if not
+if [ "$GCLOUD_SERVICE_KEY" = "" ]; then
```
Comment (keyboardsurfer) [inline scripts/ftl_setup.sh]: `if [ -z "$GCLOUD_SERVICE_KEY" ]`

File: scripts/ftl_run_tests.sh [hunk 5]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+BUILD_ID=$1
+TEST_DIR=$2
+
```
Comment (keyboardsurfer) [inline scripts/ftl_run_tests.sh]: Document

File: scripts/ftl_setup.sh (hunk 3 above)
Comment (tiembo) [reply scripts/ftl_setup.sh]: Done.

File: scripts/ftl_run_tests.sh (hunk 2 above)
Comment (tiembo) [reply scripts/ftl_run_tests.sh]: Done.

File: .circleci/config.yml (hunk 1 above)
Comment (tiembo) [reply .circleci/config.yml]: Done.

File: scripts/ftl_setup.sh (hunk 4 above)
Comment (tiembo) [reply scripts/ftl_setup.sh]: Done

**Pull request merged**